mp/src/cmake/build_info.cmake

```diff
@@ -1,31 +1,37 @@
-string(TIMESTAMP BUILD_DATE "%Y%m%d")
-string(TIMESTAMP BUILD_DATETIME UTC)
-
 find_package(Git REQUIRED)
+
 execute_process(
     COMMAND git log -1 --format=%h
     WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
     OUTPUT_VARIABLE GIT_HASH
     OUTPUT_STRIP_TRAILING_WHITESPACE
     ERROR_QUIET
 )
+string(SUBSTRING "${GIT_HASH}" 0 7 GIT_HASH)
+
 execute_process(
     COMMAND git log -1 --format=%H
     WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
     OUTPUT_VARIABLE GIT_LONGHASH
     OUTPUT_STRIP_TRAILING_WHITESPACE
     ERROR_QUIET
 )
```

Comment:
I've realized something, `%h` short hash fetch wasn't necessary, could just do `string(SUBSTRING "${GIT_LONGHASH}" 0 7 GIT_HASH)` and not be affected by `core.abbrev` git setting

Reply:
Updated